### PR TITLE
Feature/update patch version in pipeline automatically

### PR DESCRIPTION
**Pull request by DavidDamke**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -32,6 +32,7 @@
   </parent>
 
   <properties>
+    <project.version>KeepTime-1.2.0-SNAPSHOT</project.version>
```
Comment (Death111) [inline pom.xml:35]: this is not a valid version.
https://maven.apache.org/pom.html#maven-coordinates

Comment (DavidDamke) [reply pom.xml]: I removed "KeepTime" from the version because it was doubled. But I am not sure if that was the thing you meant

Comment (Death111) [reply pom.xml]: yes, that was the issue.

File: pom.xml [hunk 2]
```diff
@@ -32,6 +32,7 @@
   </parent>
 
   <properties>
+    <project.version>1.2.0-SNAPSHOT</project.version>
```
Comment (Death111) [inline pom.xml:35]: defining it as a property is interesting. not sure if there are disadvantages doing it this "variable" - looking fine i think

**Review by Death111 — APPROVED** [review]

**Pull request merged**